docs/components/integration-framework/connectors/out-of-the-box-connectors/googledrive.md

```diff
@@ -0,0 +1,200 @@
+---
+id: googledrive
+title: Google Drive connector
+description: Create empty folders or documents from templates from your BPMN process.
+---
+
+The **Google Drive connector** allows you to create empty folders or documents from templates from your BPMN process.
```

Comment:
Maybe a bit more verbose

```suggestion
The **Google Drive connector** allows you to create empty folders or documents on [Google Drive](https://drive.google.com/) from templates from your BPMN process.
```

Reply:
Is there a link we can add for `templates` (e.g., I have not used Google Drive templates) to get started with what they are?

Reply:
Added Google Drive link. For templates we don't have explicit reference, except one mentioned in FAQ section. Let me know if you think, we should add that one.